staging/src/k8s.io/code-generator/cmd/conversion-gen/args/args.go

```diff
@@ -42,6 +42,9 @@ type CustomArgs struct {
 	// generator pick up manually written conversion funcs from external packages.
 	ExtraPeerDirs []string
 
+	// CustomOutputFileBaseName is the file name for generated custom conversions.
+	CustomOutputFileBaseName string
```

Comment:
Hrm.  What does "custom" mean in this context?   For instance, ListOptions conversions wouldn't be what I would call "custom"

Reply:
Removed that - it's no longer needed.

Also renamed custom to explicit.